### PR TITLE
Refactor/change hosted connector

### PR DESCRIPTION
**Pull request by jkoenig134**

### CODE REVIEW

**Comment by Magnus-Kuhn** [comment]
The connector change looks good, but no idea about the requirement that the app is at least v4.0.0

File: _docs_integrate/integration-example.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ To do so, execute `POST /api/v2/Attributes` with the following payload:
 }
 ```
 
-You can query the Connector's address via the route `/api/v2/Account/IdentityInfo`. If you are using the Demo Connector of this Tutorial, the address is `did:e:stage.enmeshed.eu:dids:54013af2a3bffe9dc8cb99`.
+You can query the Connector's address via the route `/api/v2/Account/IdentityInfo`. If you are using the Demo Connector of this Tutorial, the address is `did:e:pilot.enmeshed.eu:dids:9364c879fcc41c87c6ecbd`.
```
Comment (vinzlange) [inline _docs_integrate/integration-example.md:83]: there must not be an empty line between the note and {: .notice--info}

**Comment by jkoenig134** [comment]
that connector is runtime v5 which we deliver with app >4

File: _docs_integrate/integration-example.md (hunk 1 above)
Comment (jkoenig134) [reply _docs_integrate/integration-example.md]: good catch! seems to be broke for a while.

**Review by vinzlange — APPROVED** [review]

**Pull request merged**